lib/dovico/model/time_entry.rb

```diff
@@ -35,13 +35,13 @@ def self.parse(hash)
     end
 
     def self.get(id)
-      entry = ApiClient.get("#{URL_PATH}/#{id}")["TimeEntries"].first
```

Comment:
I find it strange to include a trailing slash in the `URL_PATH` variable. It's better to always avoid adding trailing slashs in `path`s